src/js/App/Sidenav/index.js

```diff
@@ -1,4 +1,21 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
 import Navigation from './Navigation';
+import { Split, SplitItem } from '@patternfly/react-core/dist/esm/layouts/Split';
+import HomeIcon from '@patternfly/react-icons/dist/esm/icons/home-icon';
 
-export default () => <Navigation />;
+class SideNav extends Component {
+    onHomeButtonClick() {
+        window.location.href = `${document.baseURI}platform/landing`;
+    }
+
+    render() {
+        return (<Fragment>
+            <Split className="ins-c-navigation__header">
+                <SplitItem onClick={this.onHomeButtonClick}><HomeIcon size="md" /></SplitItem>
```

Comment:
We should either use an `a` or a `button` here for accessibility reasons

Reply:
Oh yeah, good point.